index.html

```diff
@@ -12,40 +12,171 @@
+                <div>
+                    <ul class="butto">
+                        <li><button type="button" class="ht">html</button></li>
+                        <li><button type="button" class="ssc">css</button></li>
+                        <li><button type="button" class="sj">javaScript</button></li>
+                    </ul> 
+                </div>
+                <button type="button" class="pro1">See Project</button>
+            </div>
+        </section>
+        <section class="aboutself">
+            <div class="about"><h2>About<br> Myself</h2></div>
+            <div class="par"><p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p></div>
+            <p class="connect">LET'S CONNECT</p>
+            <div>
+               <ul class="contact">
+                   <li><a href="#"><img class="github" src="images/Github@2x.png" alt="Github"></a></li>
+                   <li><a href="#"><img class="linkedin" src="images/Linkedinicon@2x.png" alt="Linkedin"></a></li>
+                   <li><a href="#"><img class="vector" src="images/Vector@2x.png" alt="Vector"></a></li>
+                   <li><a href="#"><img class="twitter" src="images/Twitter@2x.png" alt="Twitter"></a></li>
+                   <li><a href="#"><img class="email" src="images/Mail@2x.png" alt="email"></a></li>
+               </ul>
+            </div>
+            <button class="resume" type="button">Get my resume</button>
+            <ul>
+                <li>
+                    <div class="languages">
+                        <p class="language">Languages</p>
+                        <img class="dropdown" src="images2/Union.png" alt="dropdown">
+                    </div>
+                    <div class="ellipse">
+                       <img src="images2/Ellipse1.png"  alt="ellipse">&nbsp;&nbsp;<ul class="ellip"><li>JavaSript</li></ul> 
+                    </div>
+                    <div class="ellipse2">
+                        <img src="images2/Ellipse1(1).png" alt="ellipse1">&nbsp;&nbsp;
+                        <ul class="ellip"><li>HTML</li></ul>
+                    </div>
+                    <div class="ellipse3">
+                        <img src="images2/Ellipse1(2).png" alt  ="ellipse3">&nbsp;&nbsp;
+                        <ul class="ellip"><li>CSS</li></ul>   
+                    </div>
+                </li>
+                <li class="frame">
+                    <p class="skills">Frameworks</p>
+                    <img src="images2/Disabled.png" alt="arrow">
+                </li>
+                <li><hr></li>
+                <li class="frame">
+                    <p class="skills">Skills</p>
+                    <img src="images2/Disabled.png" alt="arrow">
+                </li>
+                <li><hr></li>
+            </ul>
```

Comment:
Please change the `font-weight` for `Languages`,  `Javascript`, ` HTML `, `CSS` ,`Skills` and `Frameworks` to be bold to match the Figma design.